communication/slack-config/sig-docs/docs-channels.yaml

```diff
@@ -10,3 +10,4 @@ channels:
   - name: kubernetes-docs-ko
   - name: kubernetes-docs-pt
   - name: kubernetes-docs-zh
+  - name: kubernetes-docs-vi
```

Comment:
This needs to be alphabetically sorted :)

Reply:
Oh, thank @nikhita